runtime/compiler/control/CompilationThread.cpp

```diff
@@ -3116,8 +3116,15 @@ void TR::CompilationInfo::stopCompilationThreads()
 
       fprintf(stderr, "Number of compilations per level:\n");
       for (int i = 0; i < (int)numHotnessLevels; i++)
+         {	      
          if (_statsOptLevels[i] > 0)
             fprintf(stderr, "Level=%d\tnumComp=%d\n", i, _statsOptLevels[i]);
+#if defined(J9VM_OPT_JITSERVER)
+         if (_statsRemoteOptLevels[i] > 0)
+            fprintf(stderr, "numRemoteComp=%d\n", _statsRemoteOptLevels[i]);
+#endif /* defined(J9VM_OPT_JITSERVER) */
+
+         }
```

Comment:
Somehow I think it looks nicer if the remote value appears on the same line, so maybe:
```
      for (int i = 0; i < (int)numHotnessLevels; i++)
         {
         if (_statsOptLevels[i] > 0)
            {
            fprintf(stderr, "Level=%d\tnumComp=%d", i, _statsOptLevels[i]);
#if defined(J9VM_OPT_JITSERVER)
            if (_statsRemoteOptLevels[i] > 0)
               fprintf(stderr, "\tnumRemoteComp=%d", _statsRemoteOptLevels[i]);
#endif /* defined(J9VM_OPT_JITSERVER) */
            fprintf(stderr, "\n");
            }
         }
```